boards/arm/cc2650_sensortag/doc/cc2650_sensortag.rst

```diff
@@ -0,0 +1,208 @@
+.. _cc2650_sensortag:
+
+CC2650 SensorTag
+################
+
+Overview
+********
+
+The SimpleLink CC2650 SensorTag is a Texas Instruments board based
+on the CC2650 wireless MCU. It contains multiple sensors and provides
+Bluetooth and IEEE 802.15.4 connectivity.
+
+Features
+========
+
+* CC2650 low-power wireless MCU: Bluetooth + IEEE 802.15.4 connectivity
+* Autonomous low-power sensor controller
+* Autonomous radio controller
+* Program, debug, and serial line through "DevPack" JTAG probe's USB port
+* Power from DevPack USB or batteries
+* 2 buttons and 2 LEDs for user interaction
+
+Hardware
+********
+
+The SensorTag board provides the following hardware:
+
+* CC2650 SoC in RGZ (7x7mm) 48 pins format package
+* Programmable Cortex-M3 CPU, clocked at 48MHz
+* Proprietary sensor controller
+        * 12-bit ADC, 200ks/s
+        * 2 comparators
+        * SPI + I2C digital interfaces for sensors
+        * Programmable with register-based interface
+* Cortex-M0 "RF Core" radio controller
+        * 4KB of dedicated SRAM
+        * Single transceiver, with modem and frequency synthesizer
+        * Programmable with register-based interface
+* 128KB flash
+* 20KB SRAM
+* 8KB cache
+* SPI
+* I2C
+* I2S
+* UART
+* 31 GPIOs
+* 4 32-bit timers
+* True Random Number Generator
+* Real Time Clock
+* Watchdog timer
+* 32 channels DMA controller
+* 10 sensors: ambient light, digital microphone, magnetic sensor, humidity,
+  pressure, accelerometer, gyroscope, magnetometer, object temperature and
+  ambient temperature
+
+Supported features
+==================
+
+The following SensorTag features are currently supported:
+
++-----------+------------+------------------------------------+
+| Interface | Controller | Driver/Component                   |
++===========+============+====================================+
+| NVIC      | on-chip    | Nested Vector Interrupt Controller |
++-----------+------------+------------------------------------+
+| UART      | on-chip    | serial port polling                |
++-----------+------------+------------------------------------+
+| GPIO      | on-chip    | gpio                               |
++-----------+------------+------------------------------------+
+| PINMUX    | on-chip    | pinmux                             |
++-----------+------------+------------------------------------+
+
+Connections and IOs
+===================
+
+The SensorTag has one GPIO controller, along with a flexible pin
+multiplexer. In practice, the pins are routed as described in
+:file:`boards/arm/cc2650_sensortag/board.h`; the most commonly used being:
+
++-------------------------------------------------------+
```

Comment:
This first line of the table needs the `+` for the column markers for the headings to be formatted correctly